spyder/plugins/mainmenu/plugin.py

```diff
@@ -95,50 +95,6 @@ def on_mainwindow_visible(self):
 
     # ---- Private methods
     # ------------------------------------------------------------------------
-    def _show_shortcuts(self, menu):
```

Comment:
This and the methods below were actually doing nothing. Instead, we needed to use a new functionality added in PyQt 5.10 (see the `qthelpers.create_action` changes below).